dotcom-rendering/src/components/DateTime.tsx

```diff
@@ -63,16 +65,15 @@ export const DateTime = ({
 }: Props & DisplayProps) => {
 	const { dateLocale, timeZone } = getEditionFromId(editionId);
 
-	const epoch = date.getTime();
-	const relativeTime = display === 'relative' && timeAgo(epoch);
-	const isRecent = isString(relativeTime) && relativeTime.endsWith(' ago');
+	const then = date.getTime();
+	const isRecent = display === 'relative' && Date.now() - then < ONE_WEEK;
```

Comment:
Trying to use `timeAgo` is trying to be too clever. It’s simpler to just say “less than a week old”.